src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, Countable {
+    private String color;
+    private String name;
+
+    public Figure() {
+    }
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public Figure(String color, String name) {
+        this.color = color;
+        this.name = name;
+    }
+
+    public void draw() {
+        System.out.println("Figure color: " + color + " name: " + name);
+    }
+
+    public double getArea() {

```

Comment:
I have to implement method 'getArea' also as 'draw'. Otherwise I will have to name class Figure abstract (what I can`t do since I use it`s objects) or code won`t compile